apps/web/public/static/locales/en/common.json

```diff
@@ -1110,6 +1112,7 @@
   "error_updating_settings": "Error updating settings",
   "personal_cal_url": "My personal Cal URL",
   "bio_hint": "A few sentences about yourself. this will appear on your personal url page.",
+  "timeformat_profile_hint": "Changes to this timezone will not affect your public booking pages, nor will it effect the way attendees see your booking page.",
```

Comment:
This content can probably be improved! @PeerRich  @Jaibles Any suggestions?

Reply:
What about: 
"This is an internal setting and will not affect how times are displayed on public booking pages.